sweetalert2.d.ts

```diff
@@ -360,8 +360,8 @@ declare module 'sweetalert2' {
          * @default null
          */
         input?:
-            'text' | 'email' | 'password' | 'number' | 'tel' | 'range' | 'textarea' | 'select' | 'radio' | 'checkbox' |
```

Comment:
Please do not change the current code style, at least not in the scope of this PR.

Reply:
Ok. I'll change its code block.